stl/inc/forward_list

```diff
@@ -898,19 +916,26 @@ public:
         _Insert_after(_Mypair._Myval2._Before_head(), _Val);
     }
 
+#ifdef __cpp_lib_containers_ranges
+    template <_Container_compatible_range<_Ty> _Rng>
+    void prepend_range(_Rng&& _Range) {
+        insert_range_after(before_begin(), _Range);

```

Comment:
Can we avoid the indirection and call `_Insert_range_after` directly

Reply:
Yes. And while we're at it, `_Insert_range_after` should take a `_Nodeptr` instead of an iterator, pushing iterator validation up into callers.